addon/components/frost-button.js

```diff
@@ -69,11 +74,22 @@ export default Component.extend(PropTypeMixin, {
     text: PropTypes.string,
     title: PropTypes.string,
     type: PropTypes.string,
-    vertical: PropTypes.bool
+    vertical: PropTypes.bool,
+
+    // state
```

Comment:
Originally I was all for removing comment blocks when not using them but I've found it is more of a pain if I have to go back and add them, especially if it is the long ones like `DOM Events`.